js/scripts.js

```diff
@@ -262,60 +251,60 @@ fetch(LocationApiUrl)
                 var lon = data.markets[i].coordinates.lon;
                 var latLng = new google.maps.LatLng(Lat, lon);
+                var position = i + 1;
                 nameSupermarket = data.markets[i].name;
                 addressSupermarket = data.markets[i].contact.address.street;
                 phoneSupermarket = data.markets[i].contact.phoneNumber;
                 bounds.extend(latLng);
-                // console.log(latLng);
-                createMarker(latLng, nameSupermarket, addressSupermarket, phoneSupermarket);
+                createMarker(latLng, nameSupermarket, addressSupermarket, phoneSupermarket, position);
             }
 
             map.fitBounds(bounds);
         }
     
-        function createMarker(latlng, nameSupermarket, addressSupermarket, phoneSupermarket){
+        function createMarker(latlng, nameSupermarket, addressSupermarket, phoneSupermarket, position) {
             var html = `
-            <div class="store-info-window">
-            <div class="store-info-name">
-                ${nameSupermarket}
-            </div>
-            <div class="store-info-status">
-            Open until 8:00 PM
-            </div>
-            <div class="store-info-address">
-                <div class="circle">
-                    <i class="fa fa-location-arrow"></i>
-                </div>
-                ${addressSupermarket}
-            </div>
-            <div class="store-info-phone">
-                <div class="circle">
-                    <i class="fa fa-phone"></i>
+                <div class="store-info-window">
+                    <div class="store-info-name">
+                        ${nameSupermarket}
+                    </div>
+                    <div class="store-info-status">
+                    Open until 8:00 PM
+                    </div>
+                    <div class="store-info-address">
+                        <div class="circle">
+                            <i class="fa fa-location-arrow"></i>
+                        </div>
+                        ${addressSupermarket}
+                    </div>
+                    <div class="store-info-phone">
+                        <div class="circle">
+                            <i class="fa fa-phone"></i>
+                        </div>
+                        ${phoneSupermarket}
+                    </div>
                 </div>
-                ${phoneSupermarket}
-            </div>
-        </div>
-            `;
+                `;
             
             var marker = new google.maps.Marker({
                 map: map,
                 position: latlng,
-                // label: `${i + 1}`
+                label: `${position}`
             });
 
-            google.maps.event.addListener(marker, 'click', function(){
+            google.maps.event.addListener(marker, 'click', function() {
                 infoWindow.setContent(html);
                 infoWindow.open(map, marker);
-            })
+            } )
```

Comment:
you don't need space here.
Install Prettier and configure it for your self if you still not sure what is the proper styling :)